Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -3455,7 +3455,7 @@ void CClientGame::Event_OnIngame()
 
     // This is to prevent the 'white arrows in checkpoints' bug (#274)
     CVector pos(0, 0, 0);
-    g_pGame->Get3DMarkers()->CreateMarker(87654, (e3DMarkerType)5, &pos, 1, 0.2f, 0, 0, 0, 0);
+    g_pGame->Get3DMarkers()->CreateMarker(87654, (T3DMarkerType)5, &pos, 1, 0.2f, 0, 0, 0, 0);
```

Comment:
This should use the proper enum value instead of 5

Reply:
Done